### PR TITLE
[JBWS-4233] phase 1 Removal of old obsolete links and information

### PR DESCRIPTION
**Pull request by rsearls**

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -31,17 +31,13 @@
...
-                     <li class="leaf"><a href="{{site.baseurl}}/news-history"> News History</a></li>
-                  </ul-->
                </li>
 
                <li class="open">
                   <span class="notch">&nbsp;</span>
                   <a href="{{site.baseurl}}/downloads" class="menu-title">Downloads</a>
                   <ul class="level1">
-                     <li class="leaf"><a href="{{site.baseurl}}/downloads-latest"> Latest</a></li>
+                     <li class="leaf"><a href="{{site.baseurl}}/downloads-latest"> Latest (5.x)</a></li>
```
Comment (jimma) [inline _includes/nav.html]: We have 6.x in the download page too. That's why I remove this 5.x here.

File: _includes/nav.html [hunk 2]
```diff
@@ -58,21 +54,18 @@
...
-                  <a href="/community" class="menu-title">Community</a>
+                  <a class="unlinked">Community</a>
                   <ul class="level1">
-                     <li class="leaf"><a href="https://github.com/jbossws/jbossws-cxf/discussions">Discussion</a></li>
-                     <li class="leaf"><a href="https://groups.google.com/forum/#!forum/jbossws">Forums</a></li>
-                     <li class="leaf"><a href="https://www.jboss.org/community/wiki/JBossWS">Wiki(archive)</a></li>
-                     <li class="leaf"><a href="https://community.jboss.org/en/jbossws">User Forum(archive)</a></li>
-                     <li class="leaf"><a href="https://community.jboss.org/en/jbossws/dev">Developer Forum(archive)</a></li>
+                     <li class="leaf"><a href="{{site.baseurl}}/mailinglists">Mailing Lists</a></li>
+                     <li class="leaf"><a href="https://issues.redhat.com/projects/JBWS/issues">Issue Tracker</a></li>
```
Comment (jimma) [inline _includes/nav.html]: Why the above links are removed ?

File: _includes/nav.html [hunk 3]
```diff
@@ -58,21 +54,18 @@
                </li>
                <li class="open">
                   <span class="notch">&nbsp;</span>
-                  <a href="/community" class="menu-title">Community</a>
+                  <a class="unlinked">Community</a>
```
Comment (jimma) [inline _includes/nav.html]: This is the menu button. If remove this link, it's not possible to navigate this community page.

File: _includes/nav.html [hunk 4]
```diff
@@ -58,21 +54,18 @@
...
                   </ul>
                </li>
 
                <li class="open">
                   <span class="notch">&nbsp;</span>
-                  <a href="https://issues.redhat.com/browse/JBWS" class="menu-title">Issue Tracker</a>
+                  <a href="https://jira.jboss.org/jira/browse/JBWS" class="menu-title">Issue Tracker</a>
                   <ul class="level1 jiralink">
-                     <li class="leaf"><a href="https://issues.redhat.com/browse/JBWS">JIRA</a></li>
+                     <li class="leaf"><a href="https://jira.jboss.org/jira/browse/JBWS">JIRA</a></li>
```
Comment (jimma) [inline _includes/nav.html]: The previous one is the correct JIRA link .

File: _includes/right.html [hunk 5]
```diff
@@ -1,58 +1,92 @@
...
+        </a>
+    </div>
+</div>
+
+</div><!-- end wrapper-3 -->
+
+</div><!-- end wrapper-content -->
+<div style="clear: both;"></div>
+</div><!-- end wrapper-2 -->
+</div>
```
Comment (jimma) [inline _includes/right.html:89]: This is duplicate thing I removed in the previous commit. I think we don't need to add this back.

File: build.html [hunk 6]
```diff
@@ -3,39 +3,9 @@
...
-    <li><a href="https://www.jboss.org/community/wiki/JBossWS-Mavenization">Mavenization</a></li>
-    <li><a href="https://www.jboss.org/community/wiki/JBossWS-Runningthetestsuites">Running the
-        testsuite</a></li>
-</ul>
-<p>&nbsp;</p>
-<p><strong>Further informations</strong></p>
-<p>For futher details on installation and for directions on using JBossWS-Native with JDK 6, please
-    refer to
-    <a href="https://www.jboss.org/community/wiki/JBossWS-Installation">https://www.jboss.org/community/wiki/JBossWS-Installation</a>
-</p>
```
Comment (jimma) [inline build.html:41]: Why are these things removed ?

File: docs.html [hunk 7]
```diff
@@ -3,22 +3,16 @@
...
-<p><a href="{{site.baseurl}}/documentation/5.2.4.Final/">5.2.4.Final</a> </p>
-<p><a href="{{site.baseurl}}/documentation/5.1.9.Final/">5.1.9.Final</a> </p>
 
-<!--
-<p>The <a href="https://docs.jboss.org/author/display/JBWS">documentation for the latest JBossWS
-    series</a> is available and continuously updated.</p>
-<p>Starting from JBossWS 5.0.0.Final, the release documentation is exported at
-    <a href="https://docs.jboss.org/jbossws/">https://docs.jboss.org/jbossws/</a>.</p>
-<p>&nbsp;</p>
--->
```
Comment (jimma) [inline docs.html]: Why are these documentation links removed ?

**Review by jimma — CHANGES_REQUESTED** [review]

File: _includes/nav.html (hunk 2 above)
Comment (rsearls) [reply _includes/nav.html]: These like are removed because they no longer exist.

File: _includes/nav.html (hunk 3 above)
Comment (rsearls) [reply _includes/nav.html]: None of the links exist for community.  There is nothing in Community to offer.

File: _includes/nav.html (hunk 4 above)
Comment (rsearls) [reply _includes/nav.html]: ok

File: _includes/right.html (hunk 5 above)
Comment (rsearls) [reply _includes/right.html:89]: ok

File: build.html (hunk 6 above)
Comment (rsearls) [reply build.html:41]: Link does not exist

File: docs.html (hunk 7 above)
Comment (rsearls) [reply docs.html]: Link does not exist

**Comment by rsearls** [comment]
tmp remove for on going work

**Pull request closed without merging**